packages/react-components/src/components/VideoGallery.tsx

```diff
@@ -237,8 +246,18 @@ export const VideoGallery = (props: VideoGalleryProps): JSX.Element => {
           localVideoStream?.renderElement ? (
             <>
               {
-                /* @conditional-compile-remove-from(stable) Local_Camera_switcher */
-                showCamerSwitcherInLocalPreview && <LocalVideoCameraCycleButton />
+                /* @conditional-compile-remove-from(stable) meeting/calling-composite <Local-Camera-Switcher> */
+                showCamerSwitcherInLocalPreview &&
```

Comment:
nit:
Might be cleaner to move this logic to a function like
`shouldRenderLocalVideoCameraCycleButton()`
instead of checking all the conditions inline in JSX :)

Reply:
Especially since VideoGallery is already becoming too complex to handle :D

Reply:
will do! 😊